common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst

```diff
@@ -4,8 +4,8 @@
 Mission Commands
 ================
 
-This article describes the mission commands that are supported by Copter, Plane and Rover when switched into Auto mode.
-A better list just for :ref:`Copter can be found here <copter:mission-command-list>`
+This article describes the mission commands that are supported by Copter, Plane, Sub and Rover when switched into Auto mode.
+A simpler list just for :ref:`Copter can be found here <copter:mission-command-list>`
```

Comment:
```suggestion
[site wiki="copter"]
A simpler list just for :ref:`Copter can be found here <copter:mission-command-list>`
[/site]
```
If this is Copter-only then it seems to make sense to only show for the Copter docs 🤷

Reply:
yep